scripts/build-json/build-page-json.js

```diff
@@ -116,35 +39,51 @@ async function buildPageJSON(docsPath) {
 
     // check whether this is a buildable item
     if (data && data.recipe) {
-        // build the item
-        let item = null;
-
+        let body = null;
+        let relatedContentSpec = data.related_content;
+        let contributors = null;
         let elementDirectory = docsDirectory;
 
         switch (data.recipe) {
             case 'guide':
-                item = await guide.buildGuidePageJSON(docsDirectory, data, content);
+                body = await guidePage.buildGuideContentJSON(docsDirectory, data, content);
                 // Guide pages are special because they don't have their own 
                 // directory. Instead, individual .md files share a directory. 
                 // So we need to override the name of the directory to write to.
                 elementDirectory = path.join(docsDirectory, path.basename(docsPath).split('.')[0]);
                 break;
             case 'landing-page':
-                item = await landing.buildLandingPageJSON(docsDirectory, data, content);
+                body = await landingPage.buildLandingPageJSON(docsDirectory, data, content);
                 // Landing pages are special because they don't have their own
                 // directory. Instead, individual .md files share a directory.
                 // So we need to override the name of the directory to write to.
                 elementDirectory = path.join(docsDirectory, path.basename(docsPath).split('.')[0]);
                 break;
-            case 'html-element':
-                item = await buildFromRecipe(docsDirectory, data, content);
-                break;
+            case 'html-element': {
+                    // for recipe-driven content, related_content is in the recipe
+                    const recipePath = path.join(__dirname, '..', '..', 'recipes', `${data.recipe}.yaml`);
+                    const recipe = yaml.safeLoad(fs.readFileSync(recipePath, 'utf8'));
+                    relatedContentSpec = recipe.related_content;
+                    body = await recipePage.buildRecipePageJSON(docsDirectory, data, content, recipe);
+                    // currently only reference-driven content supports contributors
+                    contributors = await packageContributors(path.join(docsDirectory, 'contributors.md'))
```

Comment:
Thank you for your note in the PR description. Yeah, this sucks. Whatever we do, soon, I'd like for the renderer to just be 1 (or 2) link to a page that lists contributors.